spec/ibandit/iban_spec.rb

```diff
@@ -164,22 +164,60 @@
         {
           country_code: "AU",
           branch_code: "123-456",
-          account_number: "123456789",
+          account_number: account_number,
         }
       end
 
-      its(:country_code) { is_expected.to eq(arg[:country_code]) }
-      its(:bank_code) { is_expected.to be_nil }
-      its(:branch_code) { is_expected.to eq("123456") }
-      its(:account_number) { is_expected.to eq("123456789") }
-      its(:swift_bank_code) { is_expected.to be_nil }
-      its(:swift_branch_code) { is_expected.to eq("123456") }
-      its(:swift_account_number) { is_expected.to eq("123456789") }
-      its(:swift_national_id) { is_expected.to eq("123456") }
-      its(:iban) { is_expected.to be_nil }
-      its(:pseudo_iban) { is_expected.to eq("AUZZ123456123456789") }
-      its(:valid?) { is_expected.to eq(true) }
-      its(:to_s) { is_expected.to eq("") }
+      context "and a 9 digit account number" do
+        let(:account_number) { "123456789" }
+
+        its(:country_code) { is_expected.to eq(arg[:country_code]) }
```

Comment:
nitpick: should we change this instead to `is_expected.to eq("AU")`? 

since we'd want this test to fail if arg[:country_code] is inadvertently mangled in a subsequent change. 

pretty much unlikely and FWIW, we'd catch it in the `its(:pseudo_iban)` check though but thought to mention it nonetheless.